src/assets/scripts/client/aircraft/FlightManagementSystem/WaypointModel.js

```diff
@@ -134,6 +133,8 @@ export default class WaypointModel {
          */
         this.isHold = false;
 
+        // TODO: This should be moved to the StandardRouteWaypointModel
```

Comment:
is this something that should be done in this PR?  Or should there be another issue for this 😉